app/datatables/repository_datatable.rb

```diff
@@ -246,40 +248,26 @@ def sort_records(records)
                               my_module_repository_rows.id IS NULL))"
           ).order("my_module_repository_rows.id NULLS #{direction}")
         else
-          sort_assigned_records(records, params[:order].values[0]['dir'])
+          sort_assigned_records(records, order_params['dir'])
         end
       elsif sorting_by_custom_column
         ci = sortable_displayed_columns[
-          params[:order].values[0][:column].to_i - 1
+          order_params['column'].to_i - 1
         ]
         column_id = @columns_mappings.key((ci.to_i + 1).to_s)
-        dir = sort_direction(params[:order].values[0])
+        dir = sort_direction(order_params)
 
         records.joins(
-          "LEFT OUTER JOIN my_module_repository_rows ON
-          (repository_rows.id = my_module_repository_rows.repository_row_id
-          AND (my_module_repository_rows.my_module_id = #{@my_module.id} OR
-                            my_module_repository_rows.id IS NULL))"
-        ).order("my_module_repository_rows.id NULLS #{direction}")
+          "LEFT OUTER JOIN (SELECT repository_cells.repository_row_id,
```

Comment:
it was something wrong with the query?
Do this affect the datatable on task?

Reply:
That was an old code, it was changed in master.